packages/core/lib/commands/config.js

```diff
@@ -21,6 +21,10 @@ const command = {
       {
         option: "set",
         description: "Set a Truffle config option value."
+      },
+      {
+        option: "list",
+        description: "List all Truffle config values",
```

Comment:
Can you add a `.` at the end of this sentence. That seems to be convention for all descriptions.